es5-persistence/src/main/java/com/netflix/conductor/dao/es5/index/ElasticSearchDAOV5.java

```diff
@@ -612,11 +612,29 @@ public void removeWorkflow(String workflowId) {
         }
     }
 
+    @Override
+    public void removeTask(String taskId) {
+        DeleteRequest request = new DeleteRequest(indexName, TASK_DOC_TYPE, taskId);
+
+        bulkRequests.putIfAbsent(TASK_DOC_TYPE, new BulkRequests(System.currentTimeMillis(), elasticSearchClient.prepareBulk()));
```

Comment:
This value is `lastFlushTime` which is used to flush request to ES every minute even if  `BulkRequest` number of requests does not reach the set upper limit in order to avoid losing writes in case of system failure.